dotnet/samples/Concepts/Agents/OpenAIAssistant_Retrieval.cs

```diff
@@ -60,13 +59,13 @@ public async Task RunAsync()
         // Local function to invoke agent and display the conversation messages.
         async Task InvokeAgentAsync(string input)
         {
-            chat.AddChatMessage(new ChatMessageContent(AuthorRole.User, input));
+            chat.Add(new ChatMessageContent(AuthorRole.User, input));
 
-            Console.WriteLine($"# {AuthorRole.User}: '{input}'");
+            this.WriteLine($"# {AuthorRole.User}: '{input}'");
```

Comment:
Bring the Console back, seems you are base in older code.

```suggestion
            Console.WriteLine($"# {AuthorRole.User}: '{input}'");
```

Reply:
Oh ok, didn't know.

Reply:
Updated